2996_reflection/reflection.md

```diff
@@ -5760,44 +5701,6 @@ Clarify in paragraph 9 that default template arguments also apply to `$splice-sp
 
 :::
 
-### [temp.arg.type]{.sref} Template type arguments {-}
-
-Extend [temp.arg.type]{.sref}/1 to cover splice template arguments:
-
-::: std
-[1]{.pnum} A `$template-argument$` for a type template parameter shall [either]{.addu} be a `$type-id$` [or a `$splice-template-argument$` whose `$splice-specifier$` designates a type]{.addu}.
-:::
-
-### [temp.arg.nontype]{.sref} Constant template arguments {-}
-
-[We don't think we have to change anything here, since if `E` is a `$splice-specifier$` that can be interpreted as a `$splice-expression$`, the requirements already fall out based on how paragraphs 1 and 3 are already worded]{.draftnote}
```

Comment:
Is it worth keeping this part for clarity? None of this section was related to splice-template-arguments before.

Reply:
I guess no, because the note was primarily driven by the desire to contrast between the changes necessary for the other two kinds.